raw-data/PH/README.md

```diff
@@ -1,18 +1,20 @@
 # PH
 
-The `PH` folder contains data drawn from journal articles, theses, and 
-published grammars, added by members of the Linguistic Phonetics 
-Laboratory at the University of Washington. The contents are described in:
+The `PH` folder contains phonological inventory data (phonemes, allophones, and tones) drawn from journal articles, theses, and published grammars. Collectively, they represent a convenience sample of languages and they were selected to improve worldwide coverage of the aggregated phoible data. Their source is tagged `phoible`.
```

Comment:
isn't the source tagged `ph` in the CSV file (not `phoible`)?

Reply:
D'oh!